src/thor/route_action.cc

```diff
@@ -549,5 +508,44 @@ void thor_worker_t::path_depart_at(Api& api, const std::string& costing) {
   }
 }
 
+/**
```

Comment:
move this method into member function. since both trace_route.cc and trace_route_action.cc use it.

Reply:
does route_action.cc use it as well?

Reply:
yes, it was originally defined in the route_action.cc. Have to move it into member functions for both of trace_route_action.cc and route_action.cc to use